app/views/applications/show.html.erb

```diff
@@ -16,6 +16,10 @@
 <%end%>
 </table>
 <h3><%= @app.status %></h3>
+<%= form_with url: "/applications/#{@app.id}", method: :patch, local: true, authenticity_token: form_authenticity_token do |form| %>
+<%= form.hidden_field :status, value: "Pending" %>
+<%= form.submit "Submit Application" %>
+<% end %>
```

Comment:
This looks good, but i think it's gonna fail to prevent a pending app from adding more pets. the search bar will still show up. You could prevent this by adding a `<% if status=in progress %> or something above the add pet section or making it hidden.